arduino-ide-extension/src/browser/contributions/check-for-updates.ts

```diff
@@ -212,7 +212,7 @@ export namespace CheckForUpdates {
     export const CHECK_FOR_UPDATES: Command = Command.toLocalizedCommand(
       {
         id: 'arduino-check-for-updates',
-        label: 'Check for Arduino Updates',
+        label: 'Check for board and library updates', //changing labelling from 'Check for Arduino updates' to 'Check for board and library updates' to adress #1646, changing translation files as well
```

Comment:
```suggestion
        label: 'Check for board and library updates',
```

This sort of "changelog" comment only pollutes the codebase. Imagine what an utter mess it would be if one of these had been added for every change made over the years. We use the Git commit history to much more effectively track revisions.

It is ironic that you put more information into this comment than you did into the commit message. I guess you didn't read the contributor guide?:

https://github.com/arduino/arduino-ide/blob/main/docs/contributor-guide/pull-requests.md#commit-guidelines